src/renate/benchmark/experiment_config.py

```diff
@@ -317,7 +318,7 @@ def train_transform(dataset_name: str) -> Optional[transforms.Compose]:
     raise ValueError(f"Unknown dataset `{dataset_name}`.")
 
 
-def test_transform(dataset_name: str) -> Optional[transforms.Normalize]:
+def test_transform(dataset_name: str) -> Optional[Union[transforms.Normalize, transforms.Compose]]:
```

Comment:
Can this be made a callable? This is very specific to existing transformations.

Reply:
Why is this also `Optional`?

Reply:
because it can return `None`